Sources/MapboxCoreNavigation/EHorizon/RoadGraph.swift

```diff
@@ -33,6 +35,8 @@ public final class RoadGraph {
         return LineString(locations.map { $0.coordinate })
     }
     
+    // MARK: Retrieving GeoJSON Geometry Of Objects
```

Comment:
In case it helps, [this tool](https://titlecaseconverter.com/) seems to do a good job of title-casing. I entered “is it the guy with the hat from Walla-Walla that had no one to pick them up?” and chose “Wikipedia” style, and it came up with something that matches Apple’s documentation well enough.

```suggestion
    // MARK: Retrieving the Shape of an Object
```